client/v3/README.md

```diff
@@ -11,6 +11,12 @@
 go get go.etcd.io/etcd/client/v3
 ```
 
+Warning: As etcd 3.5.0 was not yet released, the line above does not work. 
```

Comment:
Suggest to use `the command above` instead of `the line above` if you like it better too. Also, the CI fmt check seems failing due use of "you" so suggest to re-write something like `After first pre-release of 3.5.0 [#12498](https://github.com/etcd-io/etcd/issues/12498), etcd can be referenced as:`